react/features/invite/components/InfoDialogButton.web.js

```diff
@@ -146,7 +147,14 @@ class InfoDialogButton extends Component {
      * @returns {ReactElement}
      */
     render() {
-        const { _showDialog, _toolboxVisible, tooltipPosition } = this.props;
+        const { _toolboxVisible, tooltipPosition } = this.props;
+        let { _showDialog } = this.props;
+
+        // we do not want to show call info in iAmRecorder mode
```

Comment:
So this is to make sure it never shows even when click? What about instead not triggering the opening of the dialog on conference join?

Reply:
Yep, this is the recorder and there is nobody to click there :)

Reply:
Okay, then how about preventing the dialog from opening on conference join? Or at least making an explicit prop that generically disables the dialog from opening? config.iAmRecorder can also be taken from the redux store (features/base/config) instead of globally.